code/game/objects/items/devices/handheld_defib.dm

```diff
@@ -32,7 +38,30 @@
 		to_chat(user, "<span class='warning'>[src] is still charging!</span>")
 		return
 
-	if(emagged || (H.health <= HEALTH_THRESHOLD_CRIT) || (H.undergoing_cardiac_arrest()))
+	if(emagged)
+		if(attack_cooldown > world.time)
+			return
+
+		var/user_UID = user.UID()
+		if(HAS_TRAIT_FROM(H, TRAIT_WAS_BATONNED, user_UID)) // no following up with baton or dual wielding defibs for stunlock cheese purposes
+			return
+
+		cooldown = world.time + (cooldown)
```

Comment:
```suggestion
		cooldown = world.time + attack_cooldown
```
I think you meant to do this? Cooldown is a bool

Reply:
oh whoops, this is a leftover from something I was doing earlier that I missed - I will fix that